code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -1628,13 +1629,16 @@
 			if("Bloody Larva") // all larva are caste = blood larva
 				if(leader == "") larva_list += xenoinfo
 				larva_count++
+			if("Necromorph")
+				necromorph_count++
```

Comment:
You removed the line that generated the list of necromorph names

Reply:
Which one?

Reply:
They shouldn't appear as individual necros in the hivemind, only as X shamblers.

To avoid spamming the hivemind for important units, to keep it immersive as you'd get xeno names on necros, which makes little sense. Necros also shouldn't be wandering off alone.